packages/fast-components-styles-msft/src/utilities/elevation.ts

```diff
@@ -101,12 +100,13 @@ export function elevationShadow(
             (shadowConfig.yOffsetMultiplier * elevationValue).toFixed(1)
         );
 
-        const xOffset: string = density(xValue)(config);
-        const yOffset: string = density(yValue)(config);
-        const blur: string = density(shadowConfig.blurMultiplier * elevationValue)(
-            config
-        );
-        const opacity: number = shadowConfig.opacity;
+        const xOffset: string = toPx(xValue);
+        const yOffset: string = toPx(yValue);
+        const blur: string = toPx(shadowConfig.blurMultiplier * elevationValue);
+        const opacity: number =
+            elevationValue > 24
+                ? shadowConfig.opacity
+                : Math.round(shadowConfig.opacity * 100 * 0.6) / 100;
```

Comment:
you could just make this `shadowConfig.opacity * 60`

Reply:
I wanted to keep it at whole percentages, like opacity 0.11 instead of 0.108.